pkg/test/resourcefixture/testdata/basic/tags/v1beta1/tagstagbinding/tagbinding-project/_generated_object_tagbinding-project.golden.yaml

```diff
@@ -2,20 +2,19 @@ apiVersion: tags.cnrm.cloud.google.com/v1beta1
 kind: TagsTagBinding
 metadata:
   annotations:
+    alpha.cnrm.cloud.google.com/reconciler: direct
```

Comment:
Not for this PR, but I wonder if we should test this better now.  I feel like we can switch the controller to direct and update the golden logs, but we can still test the terraform behaviour, we just don't expect identical results on the http.log.  But we might expect that the final status is identical?